lib/gilmour/backends/redis.rb

```diff
@@ -42,8 +45,8 @@ def setup_pubsub(opts)
       @subscriber = @publisher.pubsub_client
       register_handlers
     rescue Exception => e
-      GLogger.debug e.message
```

Comment:
Till we think gilmour is not stable enough, we should set the environment variable to "debug" when we start the process. But libraries are supposed to be stable and the level should only be changed while debugging the library itself - eg em-hiredis.